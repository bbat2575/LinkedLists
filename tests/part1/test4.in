NEW 4
1
2.2
c
hello
TYPE 0
NEW 9
01
10
0000000001
1000000000
-1
-0
0000000000
-999999999
+999999999
TYPE 1
NEW 18
.0
0.
0.0
.2
0.2
2.
2.0
1.23456789
123456789.9
-5.65
+6.56
2..2
.2.2
2.2.
.2
2.
0.
.0
TYPE 2
NEW 15
a
b
c
~
`
!
@
#
$
%
^
&
*
[
\
TYPE 3
NEW 7
99                                                                                                                             
                                                                                                                             99
9.9                                                                                                                            
                                                                                                                            9.9
$                                                                                                                              
                                                                                                                              $
Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime
TYPE 4
