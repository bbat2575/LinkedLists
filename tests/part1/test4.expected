
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1 -> 2.20 -> c -> hello

> int -> float -> char -> string

> List 1: 1 -> 10 -> 1 -> 1000000000 -> -1 -> 0 -> 0 -> -999999999 -> 999999999

> int -> int -> int -> int -> int -> int -> int -> int -> int

> List 2: 0.00 -> 0.00 -> 0.00 -> 0.20 -> 0.20 -> 2.00 -> 2.00 -> 1.23 -> 123456792.00 -> -5.65 -> 6.56 -> 2..2 -> .2.2 -> 2.2. -> 0.20 -> 2.00 -> 0.00 -> 0.00

> float -> float -> float -> float -> float -> float -> float -> float -> float -> float -> float -> string -> string -> string -> float -> float -> float -> float

> List 3: a -> b -> c -> ~ -> ` -> ! -> @ -> # -> $ -> % -> ^ -> & -> * -> [ -> \

> char -> char -> char -> char -> char -> char -> char -> char -> char -> char -> char -> char -> char -> char -> char

> List 4: 99 -> 99 -> 9.90 -> 9.90 -> $                                                                                                                               ->                                                                                                                               $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> int -> int -> float -> float -> string -> string -> string

> 
